tests/testthat/test-mwrite.R

```diff
@@ -0,0 +1,207 @@
+# Copyright (C) 2013 - 2024  Metrum Research Group
+#
+# This file is part of mrgsolve.
+#
+# mrgsolve is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# mrgsolve is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with mrgsolve.  If not, see <http://www.gnu.org/licenses/>.
+
+library(testthat)
+library(mrgsolve)
+
+context("test-mwrite")
+
+test_that("convert list to code", {
+  # This could be simpler, but haven't found an easy way to do `c = "a"`
+  l <- list(a = 1, b = 1.234, c = "a", d = c("x", "y", "z"), e = FALSE)  
+  x <- mrgsolve:::tocode(l)  
```

Comment:
As far as I can tell, this is the only spot in the tests that trigger the call of `.quotes`. If it's not too much trouble, it'd be nice to also test a model that triggers that logic.